client/src/database.js

```diff
@@ -3,23 +3,56 @@ export const tastes = [
     id: '1',
     name: 'Light Roast',
     image: require('./images/light-roast.jpg'),
+    teaser: 'You will like this, if you enjoy fruity flavours and a striking acidity.',
     profile:
       'Coffees with this roast present light and fresh flavours. Nuances of berries, fruits and citrus. The taste varies from citrusy to fruity and sweet.',
+    sample: {
+      id: '1',
+      name: 'El Amanecer',
+      image: require('./images/sample-light.jpg'),
```

Comment:
Why would you use the file-loader here? Is it really necessary ? If yes, let me know why 🦆 🙂

Reply:
Hey! Thank you very much for your feedback. Well, the simple linking did not work, so I asked google and found this solution. ;) But if you have a suggestion for another workaround I'm more than happy to change that!